tests/unit/llm/test_openai_llm.py

```diff
@@ -65,10 +67,14 @@ def test_openai_llm_with_message_history_happy_path(mock_import: Mock) -> None:
     assert isinstance(res, LLMResponse)
     assert res.content == "openai chat response"
     message_history.append({"role": "user", "content": question})
-    llm.client.chat.completions.create.assert_called_once_with(  # type: ignore
-        messages=message_history,
-        model="gpt",
-    )
+    # Use assert_called_once() instead of assert_called_once_with() to avoid issues with overloaded functions
+    cast(Any, llm.client.chat.completions.create).assert_called_once()
```

Comment:
I thought these `cast(Any...)` were fixed?

Reply:
Yeah, me too. Removed and added mypy ignores in test file